app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java

```diff
@@ -39,6 +40,10 @@
     @Column(name = "operation_type")
     public String operation_type;
 
+    private static Context mContext = Home.getContext();
```

Comment:
Each time we enter this class we always come in through the same function and it already has a context passed in, we should not reach out to home to get its context, I changed this in the other branch already (this is what triggers those crashes)